jenkins_job_wrecker/modules/triggers.py

```diff
@@ -256,12 +256,65 @@ def ghprbtrigger(top, parent):
             ghpr['permit-all'] = get_bool(child.text)
         elif child.tag == 'autoCloseFailedPullRequests':
             ghpr['auto-close-on-fail'] = get_bool(child.text)
+        elif child.tag == 'blackListCommitAuthor':
+            ghpr['black-list-commit-author'] = child.text
+        elif child.tag == 'blackListLabels':
+            ghpr['black-list-labels'] = child.text.strip().split('\n')
+        elif child.tag == 'blackListTargetBranches':
+            ghpr['black-list-target-branches'] = [item[0].text for item in child]
+        elif child.tag == 'displayBuildErrorsOnDownstreamBuilds':
+            ghpr['display-build-errors-on-downstream-builds'] = True
```

Comment:
is it possible to have `<displayBuildErrorsOnDownstreamBuilds>false</displayBuildErrorsOnDownstreamBuilds>` in a config XML?

Reply:
Say no more.